Voycar.Api.Web.Tests.Integration/Members/Put_Personal/Tests.cs

```diff
@@ -0,0 +1,266 @@
+
+
+public class Tests : TestBase<App>
+{
+    private readonly App _app;
+
+    private readonly VoycarDbContext Context;
+
+
+    // Setup request client
+    public Tests(App app)
+    {
+        this._app = app;
+        this.Context = this._app.Context;
+    }
+
+
+    public R.Request CreateValidRequest()
+    {
+        return new R.Request
+        {
+            FirstName = "null",
+            LastName = "null",
+            Street = "null",
+            HouseNumber = "null",
+            PostalCode = "null",
+            City = "null",
+            Country = "null",
+            BirthDate = new DateOnly(2002, 06, 25),
+            BirthPlace = "null",
+            PhoneNumber = "null",
+        };
+    }
+
+
+    [Fact]
+    public async Task Put_Request_ReturnsOk_And_UpdateUserInDb()
+    {
+        // Arrange
+        var email = "NewMemberClient@test.de";
+        var request = this.CreateValidRequest();
+        var memberClient = await ClientFactory.CreateMemberClient(this._app, this.Context, email, "password");
+
+        // Act
+        var httpResponse = await memberClient.PUTAsync<R.Endpoint, R.Request>(request);
+
+
+        // Arrange assertion
+        var userInDb = await this.Context.Users.FirstOrDefaultAsync(user => user.Email == email.ToLowerInvariant());
+        var memberInDb = await this.Context.Members.FirstOrDefaultAsync(member => member.Id == userInDb.MemberId);
+
+        // Assert
+        httpResponse.StatusCode.Should().Be(HttpStatusCode.OK);
+        userInDb.Should().NotBeNull();
+        memberInDb.Should().NotBeNull();
+
+
+        memberInDb.FirstName.Should().Be(request.FirstName);
+        memberInDb.LastName.Should().Be(request.LastName);
+        memberInDb.Street.Should().Be(request.Street);
+        memberInDb.HouseNumber.Should().Be(request.HouseNumber);
+        memberInDb.PostalCode.Should().Be(request.PostalCode);
+        memberInDb.City.Should().Be(request.City);
+        memberInDb.Country.Should().Be(request.Country);
+        memberInDb.BirthDate.Should().Be(request.BirthDate);
+        memberInDb.BirthPlace.Should().Be(request.BirthPlace);
+        memberInDb.PhoneNumber.Should().Be(request.PhoneNumber);
+    }
+
+
+    [Fact]
+    public async Task Put_Request_JustTurned18_ReturnsOk_And_UpdateUserInDb()
+    {
+        // Arrange
+        var email = "NewMember@test.de";
+        var memberClient = await ClientFactory.CreateMemberClient(this._app, this.Context, email, "password");
+        var request = this.CreateValidRequest();
+        request.BirthDate = DateOnly.FromDateTime(DateTime.UtcNow).AddYears(-18);
+
+        // Act
+        var httpResponse = await memberClient.PUTAsync<R.Endpoint, R.Request>(request);
+
+        // Arrange assertion
+        var userInDb = await this.Context.Users.FirstOrDefaultAsync(user => user.Email == email.ToLowerInvariant());
+        var memberInDb = await this.Context.Members.FirstOrDefaultAsync(member => member.Id == userInDb.MemberId);
```

Comment:
```suggestion
        var userInDb = await this.Context.Users.FirstOrDefaultAsync(user =>
            user.Email.Equals(email, StringComparison.OrdinalIgnoreCase));
        var memberInDb = await this.Context.Members.FirstOrDefaultAsync(member =>
            member.Id == userInDb!.MemberId);
```
Selbe Suggestion wie davor.

Reply:
Funktioniert leider nicht, hatte ich auch schon probiert.